windows/src/developer/TIKE/Makefile

```diff
@@ -12,12 +12,20 @@ build: version.res manifest.res icons dirs xml
     $(COPY) tike.exe $(PROGRAM)\developer
     if exist tike.dbg $(COPY) tike.dbg $(DEBUGPATH)\developer
 
-xml:
+xml: pull-keymanweb 
     cd $(ROOT)\src\developer\tike\xml
     -del /q $(ROOT)\bin\developer\xml\*
     xcopy /s /y * $(ROOT)\bin\developer\xml
     cd $(ROOT)\src\developer\tike
 
+pull-keymanweb:
+    -rd /s/q $(ROOT)\src\developer\tike\xml\kmw\resource
```

Comment:
Correct; we use the source version of KeymanWeb in the Keyman Developer debugger.